workshops/models.py

```diff
@@ -159,6 +159,35 @@ def self_organized_workshops_per_year_remaining(self):
 #------------------------------------------------------------
 
 
+class Sponsorship(models.Model):
+    '''Represent sponsorship from a host for an event.'''
+
+    SPONSORSHIP_LEVELS = (
+        (1000, 'Level A'),
+        (5000, 'Level B'),
+    )
+
+    organization = models.ForeignKey(
+        'Host',
+        on_delete=models.CASCADE,
+        help_text='Organization sponsoring the event'
+    )
+    event = models.ForeignKey(
+        'Event',
+        on_delete=models.CASCADE,
+    )
+    level = models.PositiveIntegerField(
+        choices=SPONSORSHIP_LEVELS,
+        default=1000,
+    )
```

Comment:
My apologies, I misunderstood the question - organizational sponsorship 
for a specific workshop varies so widely that I don't think it's worth 
trying to capture as anything except a free-text field (in Notes).